buildpack.md

```diff
@@ -857,6 +858,9 @@ For each process, the buildpack:
   - A path to an executable or the file name of an executable in `$PATH`, if `args` is a list with zero or more elements.
 - MAY specify an `args` list to be passed directly to the specified executable.
 - MAY specify a `direct` boolean that bypasses the shell.
+- MAY specify a `default` boolean that indicates that the process type should be selected as the [buildpack-provided default](https://github.com/buildpacks/spec/blob/main/platform.md#outputs-4) during the export phase.
+
+An individual buildpack may only specify one process type with `default = true`. If multiple buildpacks define processes of the same type, the lifecycle MUST use the last process type definition ordered by buildpack execution for the combined process list. Therefore, a non-default process type may override a default process type, leaving the app image with no default. The lifecycle MUST select the last process type with `default = true` as the buildpack-provided default.
```

Comment:
I'm wondering if the sentence that you added in the last commit (`Therefore...`) should be even in parenthesis after the last sentence (`The lifecycle MUST...`).
I just feel that currently the `Therefore` sentence can make a confusion after reading the last sentence.

Reply:
I joined the sentences with a semi-colon, since the second is really a continuation of the first. Does that make it better?

Reply:
Updated in https://github.com/buildpacks/spec/pull/205/commits/4356d833be2d1ebf7751db87a19fd4317ec42b9d